subiquity/client/controllers/filesystem.py

```diff
@@ -236,6 +244,9 @@ async def _answers_action(self, action):
             raise Exception("could not process action {}".format(action))
 
     async def _guided_choice(self, choice):
+        if self.storage_encryption is not None:
+            self.app.next_screen(self.endpoint.guided.POST(choice))
```

Comment:
The lines below have a code path for choice==None, I suppose we should plan for that here.

Reply:
Do you mean we should error in the choice == None case? I'm not sure we'll ever support a completely custom layout for this stuff (maybe??? ) but it's certainly all a good way off.